### DISCUSSION TITLE
It works for 2 minutes and then stops working. I need help.

### DISCUSSION THREAD

**Discussion by sorz2122** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#553] [General]
Hello, I have the following problem. I followed the instructions for the BWC V2. I have an S100101 pump. At first, I used the blue LLC board, but without success. Then I tried using the red one. It worked for about 2 minutes, and then nothing works anymore. The display beeps like crazy (see the following video). I made the cables very short and also soldered a 10k resistor. The problem still persists. Can anyone help me? Did anyone else have the issue where their system worked briefly but then stopped working?
![IMG_6278](https://github.com/visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA/assets/81297488/b14646a5-ef3f-44a6-9fda-30d76b9f35f3)

https://github.com/visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA/assets/81297488/ab1828b5-8f88-40d3-af11-5a0d467ba3b9

**Comment by visualapproach** [comment]
The resistor is only for the blue LLC.

**Comment by sorz2122** [comment]
Same error without the resistor
https://github.com/visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA/assets/81297488/c8272322-a86e-4d5e-8819-64d035e42ae9

**Comment by sorz2122** [comment]
Now the circuit board is working. I replaced the Nodemcu v2 with a v3 and soldered everything again from scratch, double-checking all connections. I believe the issue was either with the v2 or a faulty solder joint
